index.html

```diff
@@ -2384,38 +2384,34 @@ <h3>Processing Errors</h3>
 below.
           </li>
           <li>
-The `code` value MUST be the integer code described in the table below
-(in parentheses, beside the type name).
-          </li>
-          <li>
-The `title` value SHOULD provide a short but specific human-readable [=string=] for
+The `title` value MUST be present and SHOULD provide a short but specific human-readable [=string=] for
```

Comment:
This should match what's in the other specs and not add a new MUST: https://w3c.github.io/vc-bitstring-status-list/#processing-errors

```suggestion
The `title` value SHOULD provide a short but specific human-readable [=string=] for
```

Reply:
Well, actually... I have added the "MUST be present and" statement exactly to be consistent with the VCDM spec, see https://www.w3.org/TR/vc-data-model-2.0/#problem-details

I do not really care either way, we have to decide whether we align with the VCDM or we align the VCDM with this...

Reply:
Right, I think we should align the VCDM with this more flexible approach (no MUSTs) that is also closer to the RFC.

Reply:
I am neutral on this. But this should be a decision of the WG, probably to be discussed on a call.

@brentzundel

Reply:
Hmm, looks like we have a consistency mismatch between VCDM, DI, and BSL :(. VCDM says each property MUST be present (this language). DI and BSL say SHOULD. RFC9457 says SHOULD as well.

I think we were trying to make the error messages useful by using MUST for certain properties existing (`type`, `title`, and `detail`). I still think that's a good idea, that is -- if you are going to raise an error, you MUST include at least those properties for the error to be immediately useful for a developer).

Let's discuss during the next WG call.

Reply:
> I think we were trying to make the error messages useful by using MUST for certain properties existing (type, title, and detail).

The `type` property is a MUST everywhere and should stay that way. It's only the properties that are advisory from the RFC (e.g., `title` and `detail`) that I believe should be SHOULDs. I think only `type` is really necessary for interoperability but these others are useful for developers -- and the fact that they are will drive whether or not they are included, i.e., the SHOULD provides flexibility but systems will probably include them for the best DX without us having to demand it.

Reply:
I'm fine w/ what @dlongley says above -- @iherman, if you're fine with that approach, can you please revert your change to MUSTs? If not, we'll have to discuss during a call.